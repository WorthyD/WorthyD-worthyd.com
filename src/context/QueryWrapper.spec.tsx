import QueryWrapper from './QueryWrapper';
import { describe, it, expect } from 'vitest';

describe('context QueryWrapper', () => {
  it('renders correctly', () => {
    expect.assertions(1);

    const ret = QueryWrapper({ element: 'Test' });

    expect(ret).toMatchSnapshot();
  });
});
