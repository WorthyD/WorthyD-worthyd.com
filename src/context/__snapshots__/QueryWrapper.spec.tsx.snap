// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`context QueryWrapper > renders correctly 1`] = `
<QueryClientProvider
  client={
    QueryClient {
      "defaultOptions": {},
      "mutationCache": MutationCache {
        "config": {},
        "listeners": [],
        "mutationId": 0,
        "mutations": [],
      },
      "mutationDefaults": [],
      "queryCache": QueryCache {
        "config": {},
        "listeners": [],
        "queries": [],
        "queriesMap": {},
      },
      "queryDefaults": [],
    }
  }
>
  Test
  <ReactQueryDevtools
    initialIsOpen={false}
  />
</QueryClientProvider>
`;
